package.json

```diff
@@ -28,46 +44,33 @@
     "vuex": "^3.5.1"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.16",
+    "@babel/eslint-parser": "^7.12.16",
     "@vue/cli-plugin-babel": "~4.5.0",
-    "@vue/cli-plugin-eslint": "^4.5.4",
+    "@vue/cli-plugin-eslint": "~5.0.0",
     "@vue/cli-plugin-router": "^4.5.4",
     "@vue/cli-plugin-unit-jest": "^4.5.6",
     "@vue/cli-service": "~4.5.0",
-    "@vue/eslint-config-prettier": "^6.0.0",
     "@vue/test-utils": "^1.0.3",
     "babel-eslint": "^10.1.0",
     "chai": "^3.5.0",
     "chai-as-promised": "^5.3.0",
     "cypress": "^9.5.2",
-    "eslint": "^6.7.2",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-cypress": "^2.12.1",
-    "eslint-plugin-prettier": "^3.1.3",
-    "eslint-plugin-vue": "^6.2.2",
-    "prettier": "^1.19.1",
```

Comment:
I started by removing every eslint related plugin and just re-added them by using `vue add eslint` and configuring it from the command line to make sure I wasn't adding any sort of extraneous package.

Reply:
Great! We should be able to remove the jest one again because that's a stale dev dependency. When I get to unit testing again I can do it with mocha which already has some tests written

Reply:
nevermind; revisit this later